src/Apps/Artwork/Components/ArtworkVideoPlayer/ArtworkVideoPlayer.tsx

```diff
@@ -12,12 +15,59 @@ interface ArtworkVideoPlayerProps extends BoxProps {
 
 const ArtworkVideoPlayer: FC<ArtworkVideoPlayerProps> = ({
   activeIndex,
-  artwork: { figures },
+  artwork: { internalID, figures, slug },
   maxHeight,
   ...rest
 }) => {
+  const [hasTrackedPlay, setHasTrackedPlay] = useState(false)
+  const { trackEvent } = useTracking()
+
   const activeVideo = figures[activeIndex]
 
+  const trackViewedVideo = () => {
+    const event: ViewedVideo = {
+      action: ActionType.viewedVideo,
+      context_screen_owner_id: internalID,
+      context_screen_owner_slug: slug,
+      context_screen_owner_type: OwnerType.artwork,
+    }
+
+    trackEvent(event)
+  }
+
+  const trackClickedPlayVideo = () => {
+    if (hasTrackedPlay) {
+      return
+    }
+
+    // Hack to detect clicks within iFrames
+    setTimeout(() => {
+      if (document.activeElement === document.querySelector("#vimeoPlayer")) {
+        setHasTrackedPlay(true)
+
+        const event: ClickedPlayVideo = {
+          action: ActionType.clickedPlayVideo,
+          context_owner_id: internalID,
+          context_owner_slug: slug,
+          context_owner_type: OwnerType.artwork,
+        }
+
+        trackEvent(event)
+      }
+    }, 0)
+  }
+
+  useEffect(() => {
+    window.addEventListener("blur", trackClickedPlayVideo)
+
+    trackViewedVideo()
+
+    return () => {
+      window.removeEventListener("blur", trackClickedPlayVideo)
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [])
```

Comment:
On mount, and only on mount, no matter what

Reply:
You need to return a teardown for that event handler or it will double bind on other pages

Reply:
Theres also no real reason to not use exhaustive deps. You're seeing this effect get called more than once?

Reply:
Well it was more: By passing those two functions in as deps I then had to wrap each in a `useCallback`, and then define the dependencies for each of those. All of that is totally irrrelevant and unnecessarily complicated if all i want to do is run an effect exactly one time, on mount

Reply:
(Can fix if you want tho, just that the linter then flagged those other functions. Either way)